index.js

```diff
@@ -0,0 +1,67 @@
+const menuButton = document.querySelector('.menu-button');
+const navmenu = document.querySelector('.drop-menu');
+const main = document.querySelector('.main-scroll');
+menuButton.addEventListener('click', () => {
+  navmenu.classList.toggle('active');
+  main.classList.toggle('active');
+  menuButton.classList.toggle('active');
+});
+
+// menu.addEventListener('click', () => {
+//   menu.classList.toggle('active');
+//   menuButton.classList.toggle('active');
+// });
```

Comment:
- [ ] To make your code clean, Kindly delete commented code.

Reply:
- Fixed Live Demo link
- Added navigation link to mobile menu for About and Home page
- Changed div tags to semantic tags (article tags) in program cards
- Removed extra h1 tags to ensure it is used once for each page 
- Made notes on CSS repetition for further projects
-  Deleted commented code to clean code
-